app/src/main/java/org/p2p/wallet/striga/user/api/response/StrigaCreateUserResponse.kt

```diff
@@ -0,0 +1,17 @@
+package org.p2p.wallet.striga.user.api.response
+
+import com.google.gson.annotations.SerializedName
+
+class StrigaCreateUserResponse(
+    @SerializedName("userId")
+    val userId: String,
+    @SerializedName("email")
+    val email: String,
+    @SerializedName("KYC")
+    val kycDetails: KycDataResponse
```

Comment:
Can check for reuse GetStrigaUserResponse?

Reply:
I've checked. After creation, kyc contains only status, if documenation telling truth